tests/plugins/emoji/dropdown.js

```diff
@@ -306,6 +306,70 @@
 					panel.hide();
 				}
 			} );
+		},
+
+		// (#2572)
+		'test translations are added to svg elements group navigation': function() {
+			if ( CKEDITOR.env.ie && !CKEDITOR.env.edge ) {
+				// Ignore test on browser which doesn't support svg.
+				assert.ignore();
+			}
+			var bot = this.editorBot;
+			bot.panel( 'EmojiPanel', function( panel ) {
+				try {
+					var doc = panel._.iframe.getFrameDocument(),
+						listItems = doc.find( '.cke_emoji-navigation_item' ),
+						translations = {
+							people: {},
+							nature: {
+								x: 3.5
+							},
+							food: {
+								y: 1
+							},
+							travel: {
+								y: 2
+							},
+							activities: {
+								x: 0.531250
+							},
+							objects: {
+								y: 2
+							},
+							symbols: {
+								y: 1
+							},
+							flags: {
+								x: 2.1933548
+							}
+						};
+
+					if ( !listItems.count() ) {
+						assert.ignore();
+					}
+
+					if ( CKEDITOR.env.edge ) {
+						// Edge round translation to 5 digits after comma (#2572).
+						translations.flags.x = Math.round( translations.flags.x * 1e5 ) / 1e5;
+					}
+
+					for ( var i = 0; i < listItems.count(); i++ ) {
+						var li = listItems.getItem( i ),
+							transformation = li.findOne( 'use' ).getAttribute( 'transform' ) || '',
+							values = transformation.match( /translate\(\s?(\d+(?:\.\d+)?)\s?(?:,?\s?(\d+(?:\.\d+)?))?\s?\)/ );
+
+						if ( !values ) {
+							assert.fail( 'There is no "transform" attribute in svg -> use element.' );
+						}
+						assert.areSame( translations[ li.data( 'cke-emoji-group' ) ].x || 0, parseFloat( values[ 1 ] || 0 ),
+							'Translation in X axis for group: "' + li.data( 'cke-emoji-group' ) + '" is incorrect.' );
+						assert.areSame( translations[ li.data( 'cke-emoji-group' ) ].y || 0, parseFloat( values[ 2 ] || 0 ),
+							'Translation in Y axis for group: "' + li.data( 'cke-emoji-group' ) + '" is incorrect.' );
+					}
+				} finally {
```

Comment:
Do we need `try/finally` blocks? Panel could be hidden before `for` loop, and it still should work, as element is living in DOM tree, it is hidden with CSS style.

Reply:
`try/finally` is used in those test to clean up after test case run
Panels are not available in any variable, that's why cleanup in `teardown` method is not possible. I consider as a good practice to clean up after a test case.
In case that test breaks, error will stop running given test case, so panel windows might remain open. This potentially could influence on further test cases. That's why I use `try/finally` to have sure that panel window will be closed even though test case fail.